src/gazebo_mavlink_interface.cpp

```diff
@@ -156,6 +156,7 @@ void GazeboMavlinkInterface::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf
   input_reference_.resize(n_out_max);
   joints_.resize(n_out_max);
   pids_.resize(n_out_max);
+  joint_max_errors_.resize(n_out_max);
```

Comment:
Would be better to have a single `vector` for those 3, but given it's already done that way, no need to change here.

Reply:
Right, makes sense.